src/Electrical/Digital/logic_vectors.jl

```diff
@@ -0,0 +1,57 @@
+import Base: size, axes, getindex, setindex!
+
+const LogicOrNumber = Union{Logic, Number}
+
+struct StdULogicVector{N} <: AbstractArray{Logic, N}
+    logic::Array{Logic}
+    level::Array{Int}
+    function StdULogicVector(l::Array)
+        N = ndims(l)
+        l = Array{Logic}(convert.(Logic, l))
+        new{N}(l, get_logic_level.(l))
+    end
+end
+
+struct StdLogicVector{N} <: AbstractArray{Logic, N}
+    logic::Array{Logic}
+    level::Array{Int}
+    function StdLogicVector(l::Array)
+        N = ndims(l)
+        l = Array{Logic}(convert.(Logic, l))
+        new{N}(l, get_logic_level.(l))
+    end
+end
+
+const LogicVector = Union{StdULogicVector, StdLogicVector}
```

Comment:
What are they used for?

Reply:
StdULogicVector, StdLogicVector are part the Std_logic_1164.
LogicVector is only for convenience while writing dispatches.